File: Python/iniciante/1116dividindoxpory.py
n = int(input())
while n:
    x, y = input().split(" ")
    x = int(x)
    y = int(y)
    if y != 0:
        print("%.1lf" % (x / y))
    else:
        print("divisao impossivel")
    n -= 1

File: C/Iniciante/1116dividindoxpory.c
#include <stdio.h>
int main() {
    int n,i,x,y;
    scanf("%d",&n);
    for(i=1;i<=n;i++){
        scanf("%i%i",&x,&y);
        if(y!=0){
            printf("%.1lf\n", (double) x / y);
        }else{
            printf("divisao impossivel\n");
        }
    }
    return 0;
}